maintenance_equipment_scrap/models/res_company.py

```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Onestein (<http://www.onestein.eu>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models
+
+
+class ResCompany(models.Model):
+    _inherit = 'res.company'
+
+    equipment_scrap_template_id = fields.Many2one(
+        'mail.template',
+        string='Equipment Scrap Email Template'
+    )
```

Comment:
I think it could be a good idea. Maybe the best is adding it on the Equipment category in order to be automatically proposed on the equipment when its category changes?